modules/op-release-notes-1-11.adoc

```diff
@@ -12,11 +12,14 @@ With this update, {pipelines-title} General Availability (GA) 1.11 is available
 
 In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.11:
 
+[NOTE]
+====
+With this update, you can use {pipelines-title} on the {product-title} cluster that runs on ARM hardware. This update includes support for the `ClusterTask` resources where images are available and the Tekton CLI tool on ARM hardware.
```

Comment:
Can we say This update also includes as both the sentences are different from each other reading it at first glance. Could you have a look. Just a suggestion.

Reply:
I removed the first instance of update. So, no need of `also`.